app/src/services/market_maker_factory.ts

```diff
@@ -40,6 +43,38 @@ class MarketMakerFactoryService {
 
     return marketMakerAddress
   }
+
+  getMarkets = async (provider: any): Promise<{ markets: Market[] }> => {
```

Comment:
I think returning `Promise<Market[]>` should be enough.

Reply:
Done